components/ratgdo/ratgdo.cpp

```diff
@@ -16,6 +16,9 @@
 
 #include "esphome/core/log.h"
 
+#define ESP_LOG1 ESP_LOGV
+#define ESP_LOG2 ESP_LOGV
```

Comment:
I added these to have more flexibility on the logging output while debugging, instead of changing in a bunch of places `ESP_LOGV` -> `ESP_LOGD`, I can just change one of both of these to `ESP_LOGD` .